main.md

```diff
@@ -257,6 +257,11 @@ described in the following, and MAY contain a holder's public key or a reference
 thereto, as well as further claims such as `iss`, `iat`, etc. as defined or
 required by the application using SD-JWTs.
 
+The SD-JWTs MUST be explicitly typed, by setting the `typ` header 
+parameter to `sd+jwt`. This is done to prevent cross-JWT 
+confusion (see "RFC8725", section 3.11).
+
+
```

Comment:
```suggestion
A `typ` parameter in the JOSE header of an SD-JWT MUST be set to the value `sd+jwt`.
```

Also can you please update a validation section with something like this?

```
 4. Validate the SD-JWT:
    1. Check that the `typ` JOSE header parameter has the value sd+jwt
    2. Ensure that a signing algorithm was used that was deemed secure for the application. Refer to [@RFC8725], Sections 3.1 and 3.2 for details.
    3. Validate the signature over the SD-JWT. 
    4. Validate the issuer of the SD-JWT and that the signing key belongs to this issuer.
    5. Check that the SD-JWT is valid using `nbf`, `iat`, and `exp` claims, if provided in the SD-JWT.
    6. Check that the claim `sd_digests` is present in the SD-JWT.
    7. Check that the `hash_alg` claim is present and its value is understand
       and the hash algorithm deemed secure.
```

Reply:
Done here
https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/83/commits/a4e455d62a6baaa879739975fd50f22256927da6

thank you, it was really important